lib/ansible/modules/cloud/openstack/os_image.py

```diff
@@ -91,6 +91,8 @@
       username: admin
       password: passme
       project_name: admin
+      os_user_domain_name: Default
```

Comment:
Can we add a note [here](https://github.com/ansible/ansible/blob/cc8e90395a9892eb58efac02ae1ac9737853d694/lib/ansible/utils/module_docs_fragments/openstack.py#L34) stating these parameters are also available ? Thanks.

Reply:
@anotherjesse Hi, would you be able to address this comment?

Reply:
@gundalow I think you had an auto-complete fail . . . @akshay1610 could you address @Akasurde's comment?

Reply:
Addressed by 6488201bf9d64575a31fb3cca9c86fad41eba361